src/test/ui/huge-array-simple-64.rs

```diff
@@ -0,0 +1,11 @@
+// ignore-64bit
```

Comment:
Shouldn't the "-64" version be the one that ignores 32bit? This seems odd.

Reply:
I got them swapped. Fixing.